operator/controllers/loki/lokistack_controller.go

```diff
@@ -123,7 +123,7 @@ type LokiStackReconciler struct {
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings;clusterroles;roles;rolebindings,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors;prometheusrules,verbs=get;list;watch;create;update;delete
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=alertmanagers,verbs=patch
-// +kubebuilder:rbac:urls=/api/v2/alerts,verbs=create
```

Comment:
If I'm not mistaken we don't need to support the old RBAC for `ocp < 4.15` because it never worked properly, but would definitely have to test

Reply:
I suggest to keep the old rights after testing this because we need to get access in older OCP version.

Reply:
Yup... we need them at least for now if the fix is not backported